### ISSUE TITLE
Annotation won't update x axis limit

### ISSUE THREAD

**Issue by gabrielziegler3** [pyannote/pyannote-audio#734]
When doing speech activity detection on multiple files, I noticed that the x axis limit is set in the first instance and is not updated in other files even though they have different duration. Should I do anything else for restarting the axes? Looking at the source code, it appears to me that it should already reset the axis at [line 349](https://github.com/pyannote/pyannote-core/blob/e647097e506c8aa0821e5cc09c8ca32f994a8c26/pyannote/core/notebook.py#L349) of the ` Notebook` class in `pyannote.core`.

Below I am calling different filepaths from a dataframe to illustrate my problem.

```python
import torch
import matplotlib.pylab as plt
from scipy.io import wavfile

def plot_audio_signal(filepath: str):
    sr, signal = wavfile.read(filepath)
    time = np.linspace(0, len(signal) / sr, num=len(signal))

    _ = plt.figure(figsize=(16, 2))
    _ = plt.title("Audio Signal")
    _ = plt.xlabel("Time (s)")
    _ = plt.ylabel("Amplitude")
    _ = plt.plot(time, signal)

i = 0
sad = torch.hub.load('pyannote/pyannote-audio', 'sad', pipeline=True)
plot_audio_signal(X["file"].iloc[i])
sad({"audio": X.iloc[i]["file"]})
```

First audio file with ~3 seconds duration.
![image](https://user-images.githubusercontent.com/18370133/131742706-dc9c3153-b221-457f-b4e1-1cfdd07430cc.png)
![image](https://user-images.githubusercontent.com/18370133/131742728-e664345b-0cc1-45bb-be7c-5bb0b7fb4b71.png)

```python
i = 3
plot_audio_signal(X["file"].iloc[i])
sad({"audio": X.iloc[i]["file"]})
```

Second audio file with ~4 seconds duration but the x limit in the annotation object is the same as previous. You can't even see where the blue line speaker stops talking.
![image](https://user-images.githubusercontent.com/18370133/131742878-11ec4564-32b4-4b04-a323-86bc56618375.png)
![image](https://user-images.githubusercontent.com/18370133/131742870-5fea3d8f-f545-4f4b-9e00-3166f65d0c59.png)

Version
```python
pyannote.audio                1.1.1
pyannote.core                 4.1
pyannote.database             4.1.1
pyannote.metrics              3.0.1
pyannote.pipeline             1.5.2
```
